ci-operator/step-registry/telco5g/cnf/tests/telco5g-cnf-tests-commands.sh

```diff
@@ -203,13 +203,31 @@ function sno_fixes {
     popd
 }
 
+function get_time_left {
+
+    now=$(date +%s)
+    then=$(cat $SHARED_DIR/start_time)
+    minutes_passed=$(( (now - then) / 60 ))
+    time_left=$(( 225 - minutes_passed ))  # the job has 4 hours to run, leave 15 minutes for must-gathering
```

Comment:
if properly using post- steps
then must-gather is accounted in the extra 1-hour you've got for tear-down
https://docs.ci.openshift.org/docs/architecture/timeouts/#ci-operator-test-process-timeouts

Reply:
Yeah, seems like it's useful for `configsuite` that runs before e2e tests and takes about 20 minutes